packages/integration-sdk-cli/src/commands/options.ts

```diff
@@ -55,19 +97,90 @@ export function getSynchronizationJobSourceOptions(
       throw new Error(
         'Invalid configuration supplied. --source api requires --scope flag.',
       );
-    synchronizeOptions.scope = options.scope;
   } else if (
     !['integration-managed', 'integration-external'].includes(options.source)
   ) {
     throw new Error(
       `Invalid configuration supplied. --source must be one of: integration-managed, integration-external, api. Received: ${options.source}.`,
     );
-  } else if (options.integrationInstanceId) {
-    synchronizeOptions.integrationInstanceId = options.integrationInstanceId;
-  } else {
+  } else if (!options.integrationInstanceId) {
     throw new Error(
       'Invalid configuration supplied. --integrationInstanceId or --source api and --scope required.',
     );
   }
-  return synchronizeOptions;
+  return options;
+}
+
+/**
+ * Returns an object containing only `SyncOptions` properties. This helps to ensure other properties of `OptionValues` are not passed to the `SynchronizationJob`.
+ */
+export function getSyncOptions(options: OptionValues): SyncOptions {
+  return {
+    source: options.source,
+    scope: options.scope,
+    integrationInstanceId: options.integrationInstanceId,
+    uploadBatchSize: options.uploadBatchSize,
+    uploadRelationshipBatchSize: options.uploadRelationshipBatchSize,
+    skipFinalize: options.skipFinalize,
+  };
+}
+
+export interface ApiClientOptions {
+  development: boolean;
+  apiBaseUrl: string;
+  apiKey?: string;
+  account?: string;
```

Comment:
Consider using `accountId`

Reply:
These reflect the names of the options and this would be updated alongside a move to `--account-id`.